--- a/extras/acme_tiny/acme_tiny.py
+++ b/extras/acme_tiny/acme_tiny.py
@@ -5,8 +5,12 @@ try:
 except ImportError:
     from urllib2 import urlopen # Python 2
 
-#DEFAULT_CA = "https://acme-staging.api.letsencrypt.org"
-DEFAULT_CA = "https://acme-v01.api.letsencrypt.org"
+DEFAULT_CA = os.environ.get('DEFAULT_CA')
+if not DEFAULT_CA:
+    if os.environ.get('LETS_ENCRYPT_STAGING'):
+        DEFAULT_CA =  "https://acme-staging.api.letsencrypt.org"
+    else:
+        DEFAULT_CA = "https://acme-v01.api.letsencrypt.org"
 
 LOGGER = logging.getLogger(__name__)
 LOGGER.addHandler(logging.StreamHandler())
